_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -0,0 +1,2544 @@
+
+class ZedRequestDependencyProvider extends SprykerZedRequestDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Client\ZedRequestExtension\Dependency\Plugin\MetaDataProviderPluginInterface>
+     */
+    protected function getMetaDataProviderPlugins(): array
+    {
+        return [
+            'currency' => new ZedRequestMetaDataProviderPlugin(),
+            'store' => new StoreMetaDataProviderPlugin(),
+            'locale' => new LocaleMetaDataProviderPlugin(),            
+        ];
+    }
+}
+
+```
+
+
+### Adjust GlueApplicationDependencyProvider class. 
+
+Remove `SetStoreCurrentLocaleBeforeActionPlugin` plugin from the `getControllerBeforeActionPlugins` method and add `StoreHttpHeaderApplicationPlugin` and `LocaleApplicationPlugin` plugins instead.
+
+```
+src/Pyz/Glue/GlueApplication/GlueApplicationDependencyProvider.php
+```
+
+```php
+<?php 
+
+namespace Pyz\Glue\GlueApplication;
+
+use Spryker\Glue\GlueApplication\GlueApplicationDependencyProvider as SprykerGlueApplicationDependencyProvider;
+use Spryker\Glue\GlueApplication\Plugin\Rest\SetStoreCurrentLocaleBeforeActionPlugin;
+use Spryker\Glue\Locale\Plugin\Application\LocaleApplicationPlugin;
+use Spryker\Glue\StoresRestApi\Plugin\GlueStorefrontApiApplication\StoreHttpHeaderApplicationPlugin;
+
+class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependencyProvider
+{
+    /**
+     * {@inheritDoc}
+     *
+     * @deprecated Will be removed without replacement.
+     *
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ControllerBeforeActionPluginInterface>
+     */
+    protected function getControllerBeforeActionPlugins(): array
+    {
+        return [
+            new StoreHttpHeaderApplicationPlugin(),
+            new LocaleApplicationPlugin(),
+        ];
+    }
+}
+```
+
+
+### Adjust GlueStorefrontApiApplicationDependencyProvider.
+
+Add `StoreHttpHeaderApplicationPlugin` plugin to the `getApplicationPlugins` method.
+
+```
+src/Pyz/Glue/GlueStorefrontApiApplication/GlueStorefrontApiApplicationDependencyProvider.php
+```
+
+```php
+<?php
+
+namespace Pyz\Glue\GlueStorefrontApiApplication;
+
+use Spryker\Glue\GlueStorefrontApiApplication\GlueStorefrontApiApplicationDependencyProvider as SprykerGlueStorefrontApiApplicationDependencyProvider;
+use Spryker\Glue\StoresRestApi\Plugin\GlueStorefrontApiApplication\StoreHttpHeaderApplicationPlugin;
+
+class GlueStorefrontApiApplicationDependencyProvider extends SprykerGlueStorefrontApiApplicationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Shared\ApplicationExtension\Dependency\Plugin\ApplicationPluginInterface>
+     */
+    protected function getApplicationPlugins(): array
+    {
+        return [
+            new StoreHttpHeaderApplicationPlugin(),
+        ];
+}
+
+```
+
+
+### Add RouterConfig class.
+
+Addd the following code to the `RouterConfig` class, use the following code:
```

Comment:
Would be nice if you can add WHY to the each of configuration section. mb projects want to get those data from other source